src/workers/postAddedSlackChannelSend.ts

```diff
@@ -61,7 +61,7 @@ export const postAddedSlackChannelSendWorker: TypedWorker<'api.v1.post-visible'>
         );
 
         if (source.private && source.type === SourceType.Squad) {
-          const admin: Pick<SourceMember, 'referralToken'> = await con
+          const admin: Pick<SourceMember, 'referralToken'> | null = await con
```

Comment:
Potential Partial candidate

Reply:
its not object syntax, we get entity or null from the db